plugins/scaffolder-backend/api-report.md

```diff
@@ -302,7 +302,10 @@ export const createPublishGitlabMergeRequestAction: (options: {
 }) => TemplateAction_2<
   {
     repoUrl: string;
-    title: string;
+    title: string
+    /**
+     * @public @deprecated use import from \@backstage/plugin-scaffolder-backend-module-github instead
```

Comment:
Not sure that this is meant to be here?

Reply:
Yeah, it looks like this was erroneously generated when I used `yarn build:api-reports`.  I've re-generated the api-reports scoping to only the package affected in this PR `yarn build:api-reports plugins/scaffolder-react`.  @benjdlambert please resolve this conversation if you feel this was sufficient.  Thanks!